openpype/tools/settings/settings/widgets.py

```diff
@@ -612,11 +612,39 @@ def mouseReleaseEvent(self, event):
         super(ProjectListView, self).mouseReleaseEvent(event)
 
 
+class ProjectListSortFilterProxy(QtCore.QSortFilterProxyModel):
+
+    def __init__(self, *args, **kwargs):
+        super(ProjectListSortFilterProxy, self).__init__(*args, **kwargs)
+        self._enable_filter = True
+
+    def filterAcceptsRow(self, source_row, source_parent):
+        if not self._enable_filter:
+            return True
+
+        index = self.sourceModel().index(source_row, 0, source_parent)
+        is_active = bool(index.data(self.filterRole()))
+        is_selected = bool(index.data(ProjectListWidget.ProjectSelectedRole))
+
+        return is_active or is_selected
+
+    def is_filter_enabled(self):
+        return self._enable_filter
+
+    def set_filter_enabled(self, value):
+        self._enable_filter = value
+        self.invalidateFilter()
+
+
 class ProjectListWidget(QtWidgets.QWidget):
     default = "< Default >"
     project_changed = QtCore.Signal()
 
-    def __init__(self, parent):
+    ProjectSortRole = QtCore.Qt.UserRole + 10
```

Comment:
Don't use came case for attributes and don't set these constants used in different code in class. It is better to define roles as constants at header part.

Reply:
Should be resolved by 0bf84ad